### ISSUE TITLE
Small error in distillation code

### ISSUE THREAD

**Issue by jamespayor** [labmlai/annotated_deep_learning_paper_implementations#66]
Just a small but relevant nitpick, I think you mean to use `output` rather than using `large_logits` again here:
https://github.com/labmlai/annotated_deep_learning_paper_implementations/blob/master/labml_nn/distillation/__init__.py#L143

**Comment by vpj** [comment]
Sorry I didn't see the issue before. It's not a small bug, it just completely stops doing the distillation.

I checked the saved experiments and it looks like I had it correct when I ran the experiments, and during the refactoring (from a single line loss calculation to target, probability calculation and then kl divergence) and documentation I copy pasted and messed it up.

Thank you very much for pointing it out. I fixed with this commit https://github.com/labmlai/annotated_deep_learning_paper_implementations/commit/e0d4b658b094546764b33e5a4349689edd435d4f
